### ISSUE TITLE
Crusade Allegiance - Suggestion

### ISSUE THREAD

**Issue by AndrewPowell124** [BSData/horus-heresy#1628]
Would it be possible to add a new allegiance with a name such as Crusade that would allow both the Loyalist and Traitor units to be chosen at the same time. This would allow pre heresy forces with characters from both sides of the heresy to be chosen in the same army.

**Comment by Mayegelt** [comment]
A difficulty with that would also be that a bunch of units also wouldn't be available. Such as Kakophoni & Gal Vorbak. Also some wargear options may not have been around. And some characters may not have been the level they are in heresy. So a praetor based named character, might have just been a consul at that point.
Best bet if you want to make a list like that is just choose traitor or loyal. Select the characters. Then swap to the other and do the same. And just live with the app telling you, you have an X should be hidden error.

**Comment by capitaladot** [comment]
I say, Andrew can write up every detail, and we can look at it?

**Comment by AndrewPowell124** [comment]
I've made a quick change. Made a Crusade allegiance which requires both the Loyalist and Traitor allegiances to be selected allowing every unit to be chosen. If it is felt that this should stop the ability for certain units such as Gal Vorbak to be chosen, I will edit the files to reflect that, but seeing as this is an unofficial change anyway I though it easier to trust players to not make these choices when building a roster. 

I also edited an issue with Abaddon I noticed where it required him to choose an armament twice.

File can be found here as GitHub was playing up a bit:

https://drive.google.com/file/d/1hB--j3eS-4Iio_sagA8YurpluO6co-xG/view?usp=sharing
